docs/adr/XXXX-timezones.md

```diff
@@ -0,0 +1,77 @@
+# timezones
+
+* Status: proposed
+* Deciders: [Sander, Wouter, Arnaud, Bart]
+* Date: 2023-01-09
+
+Technical Story: [
+    https://github.com/inmanta/inmanta-core/issues/2919,
+    https://github.com/inmanta/inmanta-core/pull/2953,
+    https://github.com/inmanta/inmanta-lsm/pull/621,
+]
+
+## Context and Problem Statement
+
+How do we deal with timezones when it comes to datetime data? This data is passed in both directions over the API, stored at
+various places in the database and manipulated throughout the code. Which invariants do we uphold with regards to explicit
+timezone information for these datetime objects?
+
+## Decision Drivers
+
+* Behavior should be consistent regardless of the timezone participating hosts (e.g. server, agent, web-console client) live in.
```

Comment:
These are bulleted lists, but referred to as numbered lists?

Reply:
I wasn't sure about the desired format. The template uses bullets, then refers to the options as `[option x]`. I think the intent is to describe the option there but that seems very verbose to me. Perhaps a numbered list is the way to go, but then we might get other ADRs that do it differently. I'm not sure how much of an issue this would be.